models/articles-model.js

```diff
@@ -0,0 +1,15 @@
+const db = require("../db/connection");
+
+exports.selectArticles = (article_id) => {
+  if (!Number(article_id)) {
+    return Promise.reject({ status: 400, msg: "Bad request" });
+  }
+  return db
+    .query(`SELECT * FROM articles WHERE article_id=${article_id}`)
```

Comment:
This query is open to sql injection and has not made use of parametric values.

Reply:
<img width="674" alt="Screenshot 2024-08-26 at 13 15 27" src="https://github.com/user-attachments/assets/6ead4e24-83a2-417a-92cf-ec463cbd55d5">

Reply:
<img width="613" alt="Screenshot 2024-08-26 at 13 16 29" src="https://github.com/user-attachments/assets/4dd69c2a-8a30-46cd-ad5f-a69a96add155">